decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -101,10 +100,10 @@ def self.newsletter_participant_ids(component)
 
         participants_has_voted_ids = Decidim::Proposals::ProposalVote.joins(:proposal).where(proposal: proposals).joins(:author).map(&:decidim_author_id).flatten.compact.uniq
 
-        endorsements_participants_ids = Decidim::Proposals::ProposalEndorsement.joins(:proposal)
-                                                                               .where(proposal: proposals)
-                                                                               .where(decidim_author_type: "Decidim::UserBaseEntity")
-                                                                               .map(&:decidim_author_id).flatten.compact.uniq
+        endorsements_participants_ids = Decidim::Endorsement.joins(:resource)
```

Comment:
```suggestion
        endorsements_participants_ids = Decidim::Endorsement
```
This joins isn't needed here anymore right?